docs-gb/kubernetes/resources/pipeline.md

```diff
@@ -2,8 +2,25 @@
 
 Pipelines allow one to connect flows of inference data transformed by `Model` components. A directed acyclic graph (DAG) of steps can be defined to join Models together. Each Model will need to be capable of receiving a V2 inference request and respond with a V2 inference response. An example Pipeline is shown below:
 
-```{literalinclude} ../../../../../../samples/pipelines/tfsimples-join.yaml
-:language: yaml
+```yaml
```

Comment:
Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

Reply:
Hi @lc525  I am not able to file the files here: https://github.com/SeldonIO/seldon-core/tree/v2/k8s/samples

Reply:
Here: https://github.com/SeldonIO/seldon-core/blob/v2/samples/pipelines/tfsimples-join.yaml

Reply:
fixed